### ISSUE TITLE
Step 4: script needs to be run on NEW exchange server

### ISSUE THREAD

**Issue by pnijjar** [MicrosoftDocs/OfficeDocs-Exchange#1971]
It was not clear to me [and apparently  to others](https://supertekboy.com/2016/03/09/exchange-2016-public-folder-migration-made-easy/) that you need to run the `PublicFoldertoMailboxMapGenerator.ps1` cmdlet on the NEW Exchange 2016 server and not the old Exchange 2010 one. Maybe I ought to have deduced this from the header title, but I did  not.

Also, I wish that each command would come with an example. I found myself looking elsewhere to see things like "what do they mean by path to the csv? Just the folder name or the `.csv` itself?"

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b6010593-c3e4-87a6-4084-27d56f23063c
* Version Independent ID: 32a2adeb-0932-3236-09d9-b7987393639b
* Content: [Use batch migration to migrate Exchange 2010 public folders to Exchange 2016](https://docs.microsoft.com/en-us/Exchange/collaboration/public-folders/batch-migration-from-previous-versions?view=exchserver-2016#feedback)
* Content Source: [Exchange/ExchangeServer/collaboration/public-folders/batch-migration-from-previous-versions.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeServer/collaboration/public-folders/batch-migration-from-previous-versions.md)
* Product: **exchange-server-it-pro**
* Technology: ****
* GitHub Login: @mattpennathe3rd
* Microsoft Alias: **v-mapenn**

**Comment by scanum** [comment]
@pnijjar  Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by get-itips** [comment]
Hello @pnijjar Paul, how are you?
Thanks for your feedback.
I tested and the script `PublicFolderToMailboxMapGenerator.ps1` can be run anywhere, (I ran it on my personal computer) as it does not include any Exchange cmdlet. you just need to input correctly the .csv file to it and then copy the resultant csv to Ex2016 for use in the next step, and for that, I think your comment is positive and I'll try to improve the article thru a pull request against it.
Thank you!
